spring-kafka/src/test/java/org/springframework/kafka/config/StreamsBuilderFactoryBeanTests.java

```diff
@@ -92,20 +92,56 @@ public void testCleanupStreams() throws IOException {
 	}
```

Comment:
Can you add your name as an `author` of this class?

Reply:
Aye, i added my name classes below.
1. `StreamsBuilderFactoryBean`
2. `StreamsBuilderFactoryBeanTests`
3. `StreamsBuilderFactoryLateConfigTests`
Please take a look 🙇‍♂️